keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -99,6 +99,7 @@ tags, and then generate with `hack/update-toc.sh`.
       - [e2e tests](#e2e-tests)
   - [Graduation Criteria](#graduation-criteria)
     - [Alpha](#alpha)
+    - [Beta](#beta)
```

Comment:
For the detailed description of step 2B - can you please update it?

Where we ended up was effectively:
- wait until watchcache is already at the requested RV (or higher)
- only then start sending events

Reply:
For step 2C - we should also cross-reference it with the KEP I linked below that shows the longterm solution for that and mention that decrease in the period is shorter-term thing only.

Reply:
Please update the section:
"Provide a fix for the long-standing issue https://github.com/kubernetes/kubernetes/issues/59848"

[we no longer have ResourceVersionMatch=MostRecent concept]

And the same for Version Skew section.

Reply:
I'm fine with separate PR, but this is the second most important thing (after PRR) now.

Reply:
ok, thx